### ISSUE TITLE
Documentation of new functions

### ISSUE THREAD

**Issue by aiorazabala** [aiorazabala/qmethod#107]
In import.q.sorts.Rd (from the latest version in `master`):

`\title{Q methodology: imports q sorts from CSV}`
should say
`\title{Q methodology: imports Q sorts from .CSV}` 
(actually, the documentation in qmethod() etc uses *.CSV. But most of all, it should be consistent at least within the help page of the same function---it doesn't stand out so much if it varies across documentation pages)

`verified q.sorts array`
should say
`verified  Q sorts array`

`All lines below the maximum Q Sort `
should say
`All lines below the maximum Q sort `

`cells contain a string ID to identify. Meaningful short handles`
meaningful short in parenthesis? I still think that this is too wordy and that just 'handle' should suffice (or any other term of your choice; it's used as a technical term defined for this specific context anyway), but this is your choice

` the q-cards by`
` the Q cards/ Q-cards by` (as in most of the standard Q method literature)

` only condition, and CSVs are expected directly under`
` only condition, and .CSV fiels are expected directly under`

`in which case items are automatically identified by full item hashes`
Are hashes different from handles and from IDs? (this is still unclear to me)
https://github.com/aiorazabala/qmethod/pull/52/files#diff-6c40e1e5d97aa249ebf9296ce4f2d37c (see comment in line 35 of man/build.q.set.Rd)

I haven't reviewed any of the other documentation, assuming that details like these may still be unresolved. Please please check it all again @maxheld83 . The emphasis on clear and consistent documentation is to help people actually use it (and to present the functions as reliable tools).

**Comment by maxheld83** [comment]
I've just tested / edited the following things for consistency:

- [ ] Q-sorts is now the universal way of referring to, well, Q-sorts (as in the original `qmethod()` functions)  
    Actually, some of the original documentation is not *completely* consistent about this either: ` \format{A list with two objects. A data frame with 9 Q sorts sorting 33 statements and a data frame with the text corresponding to the statements.}`
    Both versions can be found, I have not changed that.

**Comment by maxheld83** [comment]
@aiorazabala

> Are hashes different from handles and from IDs? (this is still unclear to me)

Hashes are *one form the IDs* can take, automatically generated by the functions
Alternatively, ID can take the form of `manual.IDs`, in which case they are, well, manual.
I'll try to find a new way to explain this, though I'm a little out of my wits.

**Comment by maxheld83** [comment]
@aiorazabala 

> meaningful short in parenthesis? I still think that this is too wordy and that just 'handle' should suffice (or any other term of your choice; it's used as a technical term defined for this specific context anyway), but this is your choice

I see.
I had intended (and streamlined) the wordier variant as a help for users, so that they won't confuse (meaning*ful*) `handle` and (meaning*less*) `ID`.
I'll take this out again then.

**Comment by maxheld83** [comment]
re: hashes vs manual.lookup: This is already in the documentation, hope it helps:

> This function offers two ways to build such unique strings (IDs), paralleled by equivalent functions in \code{\link{make.cards}}.
	\enumerate{
		\item Using the \code{manual.lookup} argument, you can specify your own, \emph{manually} created IDs, such as \code{sta102}.
		See above argument \code{manual.lookup} for how to add your own IDs.
		\item By leaving the \code{manual.lookup} argument to \code{NULL} (as is default behavior), you can use \emph{automatically} created IDs, that are computed by ``summarizing'' the \emph{complete} item wordings (\code{''Q Method is used by crazy, but charming community of ...''}) into a hexadecimal string (\code{ae128fs}), a process known as cryptographic \emph{hashing}.
		These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise identification of different versions of an item.
		This function never exposes users to the hash values; the encrypting and decrypting are done under the hood by this function and \code{\link{make.cards}}.
		See the package \href{https://github.com/aiorazabala/qmethod/wiki/Data-management}{wiki} for more information.
	}

**Comment by maxheld83** [comment]
and, @aiorazabala there is (and has been for some time) this section in `qmethod-package.Rd`:

> The import functions \code{\link{import.q.sorts}}, \code{\link{import.q.concourse}}, \code{\link{build.q.set}}, \code{\link{import.q.feedback}} and the print function \code{\link{make.cards}} imply three distinct ways to refer to \emph{items}:
  \enumerate{
    \item \emph{Full item wording}, is, obviously, the complete item, such as \code{One small community of indomitable Q-methodologists.} (...) from \code{\link{importexample}}.
      The full wording is read in from individual TEX files.
    \item The item \emph{handle} is a shorthand way of referring to an item, that should be meaningful to the researcher.
      Item handles are \emph{researcher-facing}.
      It is used as rownames in imported data, and expected as filenames for the full wording TEX files.
    \item The item \emph{ID} is another shorthand way of referring to an item, that should not be intelligble to humans, such as an arbitrary string of characters.
    Depending on the used arguments, these IDs can be either hard-entered by setting \code{manual.lookup}, or automatically generated as hashes from the full item wording (default behavior for all functions).
    Item IDs are \emph{participant-facing}.
    Item IDs are used to quickly identify items in any given Q sort, as well as to identify item feedback by participants.
    Item IDs need to be non-intelligible so as to not provide unintended, additional stimulus to participants.
  }

Hope this clarifies this a bit?
I'll add another reminder about hashes vs manual IDs in here.

**Comment by maxheld83** [comment]
ok, I have gone through all of the above issues and re-read the entire manual.
I *really* hope I got it right now.

**Comment by maxheld83** [comment]
ok, I have gone through all of the above issues and re-read the entire manual.
I *really* hope I got it right now.
